fastlane/lib/fastlane/actions/device_grid/device_grid.rb

```diff
@@ -0,0 +1,88 @@
+# rubocop:disable Style/IndentationConsistency
+# We disable this rule since we intend opening HTML labels
+module Danger
+  class Dangerfile
+    module DSL
+      # A danger plugin: https://github.com/danger/danger
+      class DeviceGrid < Plugin
+        # @param languages: Array of languages you want to see (e.g. [en-US])
+        # @param devices: Array of deviecs you want to see (e.g. ["iphone4s", "ipadair"])
+        def run(languages: nil, devices: nil)
+          public_key_path = "fastlane/public_key.txt"
+          public_key = File.read(public_key_path).strip if File.exist?(public_key_path)
+          UI.user_error!("No #{public_key_path} file found, make sure to run fastlane with `generate_device_grid` before calling `device_grid` in danger") if public_key.to_s.length == 0
+          File.delete(public_key_path)
+
+          devices ||= %w(iphone4s iphone5s iphone6s iphone6splus ipadair)
+          languages ||= ["en-US"]
+
+          deep_link_matches = pr_body.match(/:link:\s(.*)/) # :link: emoji
+          deep_link = deep_link_matches[1] if deep_link_matches
+
+          ENV["FASTLANE_DISABLE_COLORS"] = "true" # since we fetch the URL from the output
+
+          markdown("<table>")
+          languages.each do |current_language|
+            markdown("<tr>")
+              markdown("<td>")
+                markdown("<b>#{current_language[0..1]}</b>")
+              markdown("</td>")
+
+              devices.each do |current_device|
+                markdown("<td>")
+
+                params = {
+                  public_key: public_key,
+                  language: current_language,
```

Comment:
You pass `en-US` as ISO 639 (at least that 's how you documented the appetize_url_generator action API).
But it's not. `en` is.